server/controllers/duels.js

```diff
@@ -38,6 +38,9 @@ async function create(req, res) {
   try {
     const username = req.body._auth.user;
     const { opponent } = req.body;
+    const { system } = req.body;
+    const { difficulty } = req.body;
```

Comment:
On s'était mis d'accord pour utiliser la [destructuration](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#d%C3%A9composer_un_objet) pour avoir un code un peu plus concis !
```suggestion
    const { opponent, system, difficulty } = req.body;
```